contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,15 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    name_or_url:
       type: "string"
-      default: "StackStorm/st2contrib"
-    branch:
+    ref:
```

Comment:
`rev` as in revision - that's the word we're looking for. It can be particular version, hash, branch or tag.